src/RestApi.php

```diff
@@ -104,19 +108,18 @@ public static function store_api_authentication( $result ) {
 	 * @return mixed
 	 */
 	public static function maybe_init_cart_session( $return ) {
-		$wc_instance = wc();
-		// if WooCommerce instance isn't available or already have an
-		// authentication error, just return.
-		if ( ! method_exists( $wc_instance, 'initialize_session' ) || \is_wp_error( $return ) || ! self::is_request_to_store_api() ) {
+		if ( ! function_exists( 'wc_load_cart' ) || \is_wp_error( $return ) || ! self::is_request_to_store_api() ) {
 			return $return;
 		}
-		$wc_instance->frontend_includes();
-		$wc_instance->initialize_session();
-		$wc_instance->initialize_cart();
+		// @todo Core should ensure these functions load with wc_load_cart() so we don't need to manually include.
+		include_once WC_ABSPATH . 'includes/wc-cart-functions.php';
+		include_once WC_ABSPATH . 'includes/wc-notice-functions.php';
```

Comment:
Do we have an issue for this here on in Core?

Reply:
I will raise a core PR

Reply:
https://github.com/woocommerce/woocommerce/pull/26219